### PR TITLE
Kubejobs plugin tests

### PR DESCRIPTION
**Pull request by javanlacerda**

### CODE REVIEW

File: monitor/tests/plugins/kubejobs/mock_monasca.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2017 UFCG-LSD.
```
[the remark this answers is not in the document]
Comment (javanlacerda) [reply monitor/tests/plugins/kubejobs/mock_monasca.py]: done

File: monitor/tests/plugins/kubejobs/mock_k8s.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017 UFCG-LSD.
```
[the remark this answers is not in the document]
Comment (javanlacerda) [reply monitor/tests/plugins/kubejobs/mock_k8s.py]: done

**Comment by rafaelvfalc** [comment]
Add documentation to REDME.md file explaining how execute specific test class.

**Comment by rafaelvfalc** [comment]
LGTM!

**Pull request merged**